src/Suave/Compression.fs

```diff
@@ -78,14 +78,14 @@ module Compression =
     | GZIP ->
       use gzip = new GZipStream(newFileStream, CompressionMode.Compress)
       do! SocketOp.ofTask (fs.CopyToAsync gzip)
-      gzip.Close()
+      gzip.Dispose()
```

Comment:
it does I just replaced Close with Dispose without looking at the surrounding code since it was a mechanical translation.